packages/flutter/lib/src/material/ink_sparkle.dart

```diff
@@ -680,2987 +584,5 @@ class FragmentShaderManager {
   ///
   /// Words in SPIR-V are 32 bits. Every 4 elements in this list represents 1
   /// SPIR-V word. See https://www.khronos.org/registry/SPIR-V/.
-  ByteBuffer get spirvByteBuffer => Uint8List.fromList(_spirvByteList).buffer;
+  ByteBuffer get spirvByteBuffer => _spirvByteData.buffer;
```

Comment:
How confident can we be that this won't get accessed before the future completes?

How important is it to synchronously access this value?

Reply:
In particular, it lookslike the only actual callsite of this is line 524 above which is in an async method already. We should either expose this as a future or just remove it.